public/app/features/explore/Graph.tsx

```diff
@@ -79,6 +79,7 @@ interface GraphProps {
   width?: number;
   id?: string;
   range: RawTimeRange;
```

Comment:
This component surely should use TimeRange and not have to parse it?

Reply:
Yes currently fixing this